lib/events/encryptedevent.cpp

```diff
@@ -0,0 +1,28 @@
+#include "encryptedevent.h"
+
+using namespace QMatrixClient;
+
+EncryptedEvent::EncryptedEvent()
+    : RoomEvent(typeId(), matrixTypeId())
+{ }
+
+EncryptedEvent::EncryptedEvent(const QJsonObject &obj)
+    : RoomEvent(typeId(), obj)
+{ }
```

Comment:
At the very least I expect to see a quick TODO here with the note how you're going to load `_decryptedEvent` from `ciphertext`. Ideally, of course, there should be an outline of the algorithm (not necessarily working at the moment).